Running the local install fails due to node-sass

I tried installing with both NPM and Yarn.

```log
user@machine:~/Projects/esqlate$ ./esqlate --host localhost --database xxxx --user xxxx --password xxx
Server launching on 'http://localhost:3500'
Front launching on 'http://localhost:3600'

> esqlate@1.0.2 front /home/user/Projects/esqlate
> cd ./node_modules/esqlate-front && bash -c ' [ ! -d node_modules ] && npm install || true ' && npm run-script build && npm start


> esqlate@1.0.2 server /home/user/Projects/esqlate
> cd ./node_modules/esqlate-server && bash -c ' [ ! -d node_modules ] && npm install || true ' && npm run-script build && npm start


> esqlate-server@0.0.0 build /home/user/Projects/esqlate/node_modules/esqlate-server
> tsc


> esqlate-front@1.0.5 build /home/user/Projects/esqlate/node_modules/esqlate-front
> npm run-script build-templates && npm run-script build-scss && npm run-script build-ts && npm run-script build-third-party && ./node_modules/.bin/rollup -c


> esqlate-front@1.0.5 build-templates /home/user/Projects/esqlate/node_modules/esqlate-front
> mkdir -p public && cat src/index.tpl.html | sed "s^\${API_SERVER}^$API_SERVER^g" > public/index.html


> esqlate-front@1.0.5 build-scss /home/user/Projects/esqlate/node_modules/esqlate-front
> node-sass src/index.scss > public/index.css

sh: 1: node-sass: not found
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! esqlate-front@1.0.5 build-scss: `node-sass src/index.scss > public/index.css`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the esqlate-front@1.0.5 build-scss script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/user/.npm/_logs/2020-01-17T16_51_46_256Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! esqlate-front@1.0.5 build: `npm run-script build-templates && npm run-script build-scss && npm run-script build-ts && npm run-script build-third-party && ./node_modules/.bin/rollup -c`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the esqlate-front@1.0.5 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/user/.npm/_logs/2020-01-17T16_51_46_266Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! esqlate@1.0.2 front: `cd ./node_modules/esqlate-front && bash -c ' [ ! -d node_modules ] && npm install || true ' && npm run-script build && npm start`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the esqlate@1.0.2 front script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/user/.npm/_logs/2020-01-17T16_51_46_276Z-debug.log
Front finished  with exit code 1
Stopping server
src/QueryRunner.ts(2,21): error TS7016: Could not find a declaration file for module 'pg'. '/home/user/Projects/esqlate/node_modules/pg/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/pg` if it exists or add a new declaration (.d.ts) file containing `declare module 'pg';`
src/QueryRunner.ts(3,44): error TS7016: Could not find a declaration file for module 'pg'. '/home/user/Projects/esqlate/node_modules/pg/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/pg` if it exists or add a new declaration (.d.ts) file containing `declare module 'pg';`
src/QueryRunner.ts(128,40): error TS7006: Parameter 'f' implicitly has an 'any' type.
src/QueryRunner.ts(183,17): error TS7031: Binding element 'rows' implicitly has an 'any' type.
src/middleware.ts(2,49): error TS7016: Could not find a declaration file for module 'express'. '/home/user/Projects/esqlate/node_modules/express/index.js' implicitly has an 'any' type.
  Try `npm install @types/express` if it exists or add a new declaration (.d.ts) file containing `declare module 'express';`
src/middleware.ts(4,19): error TS7016: Could not find a declaration file for module 'json5'. '/home/user/Projects/esqlate/node_modules/json5/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/json5` if it exists or add a new declaration (.d.ts) file containing `declare module 'json5';`
src/middleware.ts(371,37): error TS7006: Parameter 'err' implicitly has an 'any' type.
src/nextWrap.ts(1,87): error TS7016: Could not find a declaration file for module 'express'. '/home/user/Projects/esqlate/node_modules/express/index.js' implicitly has an 'any' type.
  Try `npm install @types/express` if it exists or add a new declaration (.d.ts) file containing `declare module 'express';`
src/persistence.ts(9,27): error TS7016: Could not find a declaration file for module 'json2csv'. '/home/user/Projects/esqlate/node_modules/json2csv/dist/json2csv.cjs.js' implicitly has an 'any' type.
  Try `npm install @types/json2csv` if it exists or add a new declaration (.d.ts) file containing `declare module 'json2csv';`
src/server.ts(3,24): error TS7016: Could not find a declaration file for module 'body-parser'. '/home/user/Projects/esqlate/node_modules/body-parser/index.js' implicitly has an 'any' type.
  Try `npm install @types/body-parser` if it exists or add a new declaration (.d.ts) file containing `declare module 'body-parser';`
src/server.ts(4,33): error TS7016: Could not find a declaration file for module 'cors'. '/home/user/Projects/esqlate/node_modules/cors/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/cors` if it exists or add a new declaration (.d.ts) file containing `declare module 'cors';`
src/server.ts(5,92): error TS7016: Could not find a declaration file for module 'express'. '/home/user/Projects/esqlate/node_modules/express/index.js' implicitly has an 'any' type.
  Try `npm install @types/express` if it exists or add a new declaration (.d.ts) file containing `declare module 'express';`
src/server.ts(8,22): error TS7016: Could not find a declaration file for module 'pg'. '/home/user/Projects/esqlate/node_modules/pg/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/pg` if it exists or add a new declaration (.d.ts) file containing `declare module 'pg';`
src/server.ts(14,19): error TS7016: Could not find a declaration file for module 'json5'. '/home/user/Projects/esqlate/node_modules/json5/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/json5` if it exists or add a new declaration (.d.ts) file containing `declare module 'json5';`
src/server.ts(210,47): error TS7006: Parameter '_req' implicitly has an 'any' type.
src/server.ts(210,53): error TS7006: Parameter 'res' implicitly has an 'any' type.
src/server.ts(210,58): error TS7006: Parameter 'next' implicitly has an 'any' type.
src/server.ts(315,19): error TS7006: Parameter 'e' implicitly has an 'any' type.
src/server.ts(319,12): error TS7006: Parameter 'lookupOid' implicitly has an 'any' type.
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! esqlate-server@0.0.0 build: `tsc`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the esqlate-server@0.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/user/.npm/_logs/2020-01-17T16_51_49_394Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! esqlate@1.0.2 server: `cd ./node_modules/esqlate-server && bash -c ' [ ! -d node_modules ] && npm install || true ' && npm run-script build && npm start`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the esqlate@1.0.2 server script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/user/.npm/_logs/2020-01-17T16_51_49_403Z-debug.log
```
</details>

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,0 +1,3 @@
+language: node_js
+node_js:
+ - "12"

--- a/package.json
+++ b/package.json
@@ -1,11 +1,14 @@
 {
   "name": "esqlate",
-  "version": "1.0.2",
+  "version": "1.0.3",
   "description": "The front page of the Esqlate project ",
   "main": "runner.js",
   "scripts": {
-    "front": "cd ./node_modules/esqlate-front && bash -c ' [ ! -d node_modules ] && npm install || true ' && npm run-script build && npm start",
-    "server": "cd ./node_modules/esqlate-server && bash -c ' [ ! -d node_modules ] && npm install || true ' && npm run-script build && npm start"
+    "front-prepare": "cd ./node_modules/esqlate-front && bash -c ' [ ! -d node_modules ] && npm install || true ' && npm run-script build-templates",
+    "server-prepare": "cd ./node_modules/esqlate-server && bash -c ' [ ! -d node_modules ] && npm install || true '",
+    "front": "npm run-script front-prepare && cd ./node_modules/esqlate-front && npm start",
+    "server": "npm run-script server-prepare && cd ./node_modules/esqlate-server && npm start",
+    "test": "npm run-script front-prepare && npm run-script server-prepare"
   },
   "repository": {
     "type": "git",
@@ -18,8 +21,8 @@
   },
   "homepage": "https://github.com/forbesmyester/esqlate#readme",
   "dependencies": {
-    "esqlate-front": "^1.0.6",
-    "esqlate-server": "0.0.0",
+    "esqlate-front": "^1.0.7",
+    "esqlate-server": "^1.0.0",
     "get-port": "^5.1.0",
     "open": "^7.0.0",
     "yargs": "^15.1.0"

